throttler_test.go

```diff
@@ -178,18 +178,19 @@ func TestBatchedThrottler(t *testing.T) {
 		th := NewBatchedThrottler(test.MaxWorkers, len(test.ToBeBatched), test.BatchSize)
 		for i := 0; i < th.TotalJobs(); i++ {
 			go func(tbbSlice []string, expectedSlice []string) {
+				var err error
 				if !reflect.DeepEqual(tbbSlice, expectedSlice) {
-					t.Fatalf("wanted: %#v | got: %#v", expectedSlice, tbbSlice)
+					err = fmt.Errorf("wanted: %#v | got: %#v", expectedSlice, tbbSlice)
```

Comment:
Good catch on the test.

Reply:
Not going to lie, that was `golint` being smart, not me. :)